extensions.json

```diff
@@ -521,16 +521,16 @@
     "custom": [
       "npx pnpm install --no-frozen-lockfile --shamefully-hoist",
       "npx pnpm run build",
-      "cd ./extensions/vscode-vue-language-features && npx pnpm run prepack:browser && npx vsce package -o extension.vsix"
+      "cd ./extensions/vscode-vue-language-features && npx pnpm run prepack:node && npx vsce package -o extension.vsix"
     ]
   },
-  "johnsoncodehk.vscode-typescript-vue-plugin": {
+  "Vue.vscode-typescript-vue-plugin": {
     "repository": "https://github.com/johnsoncodehk/volar",
     "location": "extensions/vscode-typescript-vue-plugin",
     "custom": [
-      "npx pnpm install --no-frozen-lockfile --shamefully-hoist",
+      "npx pnpm install",
       "npx pnpm run build",
-      "cd ./extensions/vscode-typescript-vue-plugin && npx vsce package -o extension.vsix"
+      "cd ./extensions/vscode-typescript-vue-plugin && npm install && npx vsce package -o extension.vsix"
```

Comment:
I think `npm install &&` is redundant here, you should install by pnpm in root dir.